src/controller/python/chip/CertificateAuthority.py

```diff
@@ -92,7 +92,7 @@ def __init__(self, chipStack: ChipStack.ChipStack, caIndex: int, persistentStora
             raise ValueError("Encountered error initializing OpCreds adapter")
 
         self._isActive = True
-        self._activeAdmins = []
+        self._activeAdmins: list[FabricAdmin.FabricAdmin] = []
```

Comment:
the new way is `list` without Capital, using generics instead of the typing module, but it wouldn't work with python 3.8 (minimum is 3.9) --> ill make it `List` just in case